labrad/wrappers.py

```diff
@@ -390,18 +390,76 @@ def _send(self, *args, **kw):
         return self._cxn._send(self.ID, *args, **kw)
 
 @inlineCallbacks
-def getConnection(host=C.MANAGER_HOST, port=C.MANAGER_PORT, name="Python Client", password=None):
+def getConnection(host=C.MANAGER_HOST, port=None, name="Python Client", password=None, tls=C.MANAGER_TLS):
     """Connect to LabRAD and return a deferred that fires the protocol object."""
-    p = yield protocol.factory.connectTCP(host, port, C.TIMEOUT)
+    if port is None:
+        port = C.MANAGER_PORT_TLS if tls.lower() == 'on' else C.MANAGER_PORT
```

Comment:
I'd pull out
use_tls = tls.lower() == 'on'
above here and reuse use_tls here and below.

tls.lower() == 'off' is also checked below, that means there are 3 states for tls:
- 'on'
- 'off'
- 'something else'
maybe use a boolean value or just check tls.lower() == 'on' and tls.lower() != 'on' ?